h2o-py/h2o/explanation/_explain.py

```diff
@@ -1930,6 +1930,17 @@ def _get_xy(model):
     return x, y
 
 
+def _get_treatment(model):
+    # type: (h2o.model.ModelBase) ->  str
+    """
+    Get treatment column.
+    :param model: H2O Model
+    :returns: treatment column name
+    """
+    treat = model.actual_params["treatment_column"]
```

Comment:
I'd prefer not to raise a KeyError if there's no treatment column in the actual_params.
```suggestion
    treat = model.actual_params.get("treatment_column")
```